courses/views/v2/__init__.py

```diff
@@ -598,3 +601,39 @@ def destroy(self, request, pk=None):
         )
 
         return self.list(request)
+
+
+@extend_schema(
+    description="Returns the json for the verifiable credential with the given ID, if it belongs to the user.",
```

Comment:
I think right now this allows unauthenticated access, should we change the description?
```suggestion
    description="Returns the json for the verifiable credential with the given ID.",
```